src/solvers/dgsem_tree/containers.jl

```diff
@@ -28,9 +28,11 @@ function reinitialize_containers!(mesh::TreeMesh, equations, dg::DGSEM, cache)
     init_boundaries!(boundaries, elements, mesh)
 
     # re-initialize mortars container
-    @unpack mortars = cache
-    resize!(mortars, count_required_mortars(mesh, leaf_cell_ids))
-    init_mortars!(mortars, elements, mesh)
+    if hasproperty(cache, :mortars) # cache_parabolic does not carry mortars
```

Comment:
`if` avoids here essentially a different version of `reinitialize_containers!`.
In case we would like to avoid the `if`-clause, one could add `function reinitialize_containers!(mesh::TreeMesh, equations::AbstractEquationsParabolic, dg::DGSEM, cache)` or similar